src/sentry/tasks/reports.py

```diff
@@ -229,25 +230,72 @@ def build_project_series(start__stop, project):
     assert resolution == rollup, "resolution does not match requested value"
 
     clean = partial(clean_series, start, stop, rollup)
+
+    def zerofill_clean(data):
+        return clean(zerofill(data, start, stop, rollup, fill_default=0))
+
     issue_ids = project.group_set.filter(
         status=GroupStatus.RESOLVED, resolved_at__gte=start, resolved_at__lt=stop
     ).values_list("id", flat=True)
 
+    # TODO: The TSDB calls could be replaced with a SnQL call here
```

Comment:
I didn't make much effort to move the group error counting over to a snql query, though it probably wouldn't be terribly hard.